pcdsdevices/state.py

```diff
@@ -304,7 +309,7 @@ def _create_states_enum(self):
                 for alias in aliases:
                     state_def[alias] = i
         enum_name = self.__class__.__name__ + 'States'
-        enum = Enum(enum_name, state_def, start=0)
+        enum = HelpfulIntEnum(enum_name, state_def, start=0, module=__name__)
```

Comment:
TIL the enum type constructor has a `module` arg

Reply:
Same for me - the ["Warning!"](https://docs.python.org/3.11/howto/enum.html#functional-api) brought it to my attention - unpickleable without it:
<img width="650" alt="image" src="https://user-images.githubusercontent.com/5139267/152251809-e6b3ab96-dc08-4f4a-85cd-17378ff1b738.png">

Reply:
Huh, interesting